packages/pf3-component-mapper/src/form-fields/button.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
-import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { Button as PFButton } from 'patternfly-react';
 
-const ButtonOverride = (props) => <Button style={{ marginLeft: 3 }} { ...props } />;
+export const Button = ({ label, variant, className, ...rest }) => {
+  let { formOptions, FieldProvider, validate, ...props } = { ...rest };
+  return <PFButton bsStyle={ variant } className={ className } { ...props }>{ label }</PFButton>;
+};
 
-export default ButtonOverride;
+Button.propTypes = {
+  label: PropTypes.string.isRequired,
+  variant: PropTypes.string,
+  className: PropTypes.string,
+};
+
+Button.defaultProps = {
+  variant: undefined,
```

Comment:
This default value is not required. (We don't check it via linter so no need for `undefined` = `undefined`)